style/style.css

```
@@ -0,0 +1,271 @@
+* {
+    box-sizing: border-box;
+}
+/* 
+@media (min-width: 1920px) {
+   
+} */
+
+html {
+    /* word-break: keep-all; */
+    font-family: Pretendard, sans-serif;
+    
+}
+
+body {
+    width: 100%;
+    height: 100%;
+    /* max-width:1920px; */
+    margin: 0;
+    background: #FCFCFC;
+    color: #fff;
+}
+
+.wrap {
+    height:100%;
+    max-width: 1920px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    align-self: stretch;
+    gap:auto;
+}
+
+
+
+header {
+    display: flex;
+    width: 100%;
+    height: 70px;
+    padding: 9px 400px;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    flex-shrink: 0;
+
+    border-bottom: 1px solid #DFDFDF;
+    background: #FFF;
+}
+
+
+.logo {
+    display: flex;
+    width: 116px;
+    align-items: flex-start;
+    gap: 12px;
+    cursor: pointer;
+}
+
+.loginbutton {
+    display: flex;
+    width: 128px;
+    height: 48px;
+    padding: 12px 23px;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+
+    border-radius: 8px;
+    background: #3692FF;
+    color: #F3F4F6;
+
+    cursor: pointer;
+
+    font-family: Pretendard;
+    font-size: 16px;
+    font-style: normal;
```

P2:
font family가 반복적으로 적용될 필요가 없을 것 같아요. font-style도 작성하지 않으시면 자동으로 font-style: normal이 적용되니 불필요할 것 같습니다.

```suggestion
    font-size: 16px;
```